sklearn/decomposition/_factor_analysis.py

```diff
@@ -204,6 +200,12 @@ def fit(self, X, y=None):
         self : object
             FactorAnalysis class instance.
         """
+
+        if self.svd_method not in ["lapack", "randomized"]:
+            raise ValueError(
+                f"SVD method {self.svd_method!r} is not supported. Please consider the documentation"
```

Comment:
This line is too long (> 88 characters). We usually specify the potential choice instead of referring to the documentation:

```suggestion
               f"SVD method {self.svd_method!r} is not supported. Possible methods "
                "are either 'lapack' or 'randomized'."
```